pkgs/games/prismlauncher/default.nix

```diff
@@ -67,7 +86,7 @@ stdenv.mkDerivation rec {
       their own mods, texture packs, saves, etc) and helps you manage them and
       their associated options with a simple interface.
     '';
-    platforms = platforms.linux;
+    platforms = with platforms; linux ++ darwin;
```

Comment:
`platforms.unix` maybe?

Reply:
i'm not sure if that's the best idea without knowing if the current expression will actually build on those other platforms - mainly bsd and cygwin. definitely something to look into for a future pr though

Reply:
I just checked other packages and some of them also do it like this. Fine by me then :D